llvm/docs/DeveloperPolicy.rst

```diff
@@ -1154,6 +1154,13 @@ acceptable for their contributions.  We feel that a high burden for relicensing
 is good for the project, because contributors do not have to fear that their
 code will be used in a way with which they disagree.
 
+Embedded Copyright Statements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The LLVM project does not accept contributions that include in-source copyright
+notices except where such notices are part of a larger external project being
+added as a vendored dependency.
```

Comment:
I'd suggest adding a sentence of rationale: "LLVM source code lives for a long time and is edited by many people, the best way to track contributions is through revision control history."